app/apps/server/bridges/scheduler.js

```diff
@@ -1,23 +1,44 @@
 import Agenda from 'agenda';
 import { MongoInternals } from 'meteor/mongo';
+import { StartupType } from '@rocket.chat/apps-engine/definition/scheduler';
 
 function _callProcessor(processor) {
-	return (job) => processor(job.attrs.data);
+	return (job) => processor(job?.attrs?.data || {});
 }
 
 export class AppSchedulerBridge {
 	constructor(orch) {
 		this.orch = orch;
 		this.scheduler = new Agenda({
 			mongo: MongoInternals.defaultRemoteCollectionDriver().mongo.client.db(),
-			collection: 'rocketchat_agenda_jobs',
+			defaultConcurrency: 1,
```

Comment:
What is the reasoning for lowering the `defaultConcurrency`?

Reply:
For ensuring only one instance of a given job is run at a time. Covers the recurrent jobs. Without this, when I had multiple instances of RC, the same job was being run everywhere at the same time instead of just in one place

Reply:
Oh, alright! Thanks for clarifying :) Maybe it's worth adding the comment

```suggestion
                        // this ensures the same job doesn't get executed multiple times in a cluster 
			defaultConcurrency: 1,
```